app/frontend/wallet/shelley-wallet.ts

```diff
@@ -160,12 +152,13 @@ const MyAddresses = ({accountIndex, cryptoProvider, gapLimit, blockchainExplorer
 }
 
 const ShelleyBlockchainExplorer = (config) => {
+  // TODO: move to separate file
   const be = BlockchainExplorer(config)
 
   const fixAddress = (address) => (isShelleyAddress(address) ? bechAddressToHex(address) : address)
   const fix = (addresses: Array<string>): Array<string> => {
     return addresses.map(fixAddress)
-  }
+  } // TODO: rename to better name than "fix"
```

Comment:
it would probably be more idiomatic to use "compose" helper:

`const compose = (...fns) => x => fns.reduceRight((v, f) => f(v), x);`

and then you can write the exported functions using it as compose(fn, transformShelleyAddresses)

source: https://medium.com/javascript-scene/master-the-javascript-interview-what-is-function-composition-20dfb109a1a0